### PR TITLE
Add 'tail -f' functionality for job resource fetching

### PR DESCRIPTION
**Pull request by jdemaeyer**
Implements #91

### CODE REVIEW

File: shub/utils.py [hunk 1]
```diff
@@ -270,3 +272,30 @@ def get_sources(use_closest=True):
...
+    returns its key. By default, the key is retrieved via
+    json.loads(item)['_key'] (suitable for the iter_json iterators).
+
+    As a handy shortcut, iter_func will be iterated through only once if
+    `follow` is set to `False`.
+    """
+    if not follow:
+        for item in iter_func():
+            yield item
+        raise StopIteration
```
Comment (dangra) [inline shub/utils.py]: why raising StopIteration instead of `return`?

Comment (jdemaeyer) [inline shub/utils.py]: Ah, thanks, I forgot that `return` without value was allowed in generators.

**Comment by dangra** [comment]
hey, the appveyor failure is my fault. I've disabled it now.

**Comment by pablohoffman** [comment]
LGTM, needs rebase

**Comment by jdemaeyer** [comment]
thanks, rebased

**Pull request merged**